basic_client.go

```diff
@@ -90,35 +89,10 @@ func NewBasicConn(lndHost, tlsPath, macDir, network string,
 		grpc.WithTransportCredentials(creds),
 	}
 
-	if macDir == "" {
-		macDir = filepath.Join(
-			defaultLndDir, defaultDataDir, defaultChainSubDir,
-			"bitcoin", network,
-		)
-	}
-
-	// Starting with the set of default options, we'll apply any specified
-	// functional options to the basic client.
-	bco := defaultBasicClientOptions()
-	bco.applyBasicClientOptions(basicOptions...)
-
-	macPath := filepath.Join(macDir, bco.macFilename)
-
-	// Load the specified macaroon file.
-	macBytes, err := ioutil.ReadFile(macPath)
-	if err == nil {
-		// Only if file is found
-		mac := &macaroon.Macaroon{}
-		if err = mac.UnmarshalBinary(macBytes); err != nil {
-			return nil, fmt.Errorf("unable to decode macaroon: %v",
-				err)
-		}
-
-		// Now we append the macaroon credentials to the dial options.
-		cred := macaroons.NewMacaroonCredential(mac)
-		opts = append(opts, grpc.WithPerRPCCredentials(cred))
-		opts = append(opts, grpc.WithDefaultCallOptions(maxMsgRecvSize))
-	}
+	// Now we append the macaroon credentials to the dial options.
+	cred := macaroons.NewMacaroonCredential(mac)
+	opts = append(opts, grpc.WithPerRPCCredentials(cred))
```

Comment:
Can add these two calls directly to the initialization of the `opts` variable above, then no `append()` is needed.